app_settings.py

```diff
@@ -7,8 +7,9 @@
 from wam import settings  # pylint: disable=import-error
 
 # add the package leaflet to the wam core project installed apps
-if os.name == 'posix':
-    settings.INSTALLED_APPS.append('leaflet')
+#if os.name == 'posix':
+#    if 'leaflet' not in settings.INSTALLED_APPS:
+#        settings.INSTALLED_APPS.append('leaflet')
 
```

Comment:
You can delete those lines instead of commenting them :)

Reply:
The comment is what causes the errors in Travis, so this will fix the errors as well ;)